### DISCUSSION TITLE
Is there any way to fetch bytes and filename from uploaded files by using multipart without using TempFile?

### DISCUSSION THREAD

**Discussion by hrekov** [rwf2/Rocket#1946] [Questions]
Hi! From docs I saw that I can receive file from multipart form by using TempFile<'_>, but is there any way to fetch raw bytes and filename directly from request without using temporary file?

**Comment by daaku** [comment]
Been trying to figure out the same thing! I would be happy to just have the bytes in a `Vec<u8>` or `&[u8]` for my use case.
